src/io/appium/android/apis/view/TextFields.java

```diff
@@ -20,22 +20,41 @@
 // class is in a sub-package.
 import io.appium.android.apis.R;
 
+import java.lang.CharSequence;
+
 import android.app.Activity;
 import android.os.Bundle;
-import android.widget.ArrayAdapter;
-import android.widget.Button;
-import android.widget.Spinner;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.widget.TextView;
+import android.widget.EditText;
 
 
 /**
  * A gallery of basic controls: Button, EditText, RadioButton, Checkbox,
  * Spinner. This example uses the light theme.
  */
 public class TextFields extends Activity {
-
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.text_fields);
+
+        EditText passwordInput = (EditText)findViewById(R.id.edit1);
+        final TextView passwordOutput = (TextView)findViewById(R.id.edit1Text);
+        passwordInput.addTextChangedListener(new TextWatcher () {
+            @Override
+            public void afterTextChanged (Editable s) {
```

Comment:
are we intentionally removing default functionality by overriding and leaving these empty?